src/store/state.ts

```diff
@@ -110,6 +110,7 @@ export interface VQBState {
    * variable delimiter for templating
    */
   variableDelimiters?: VariableDelimiters;
+  featureFlags: any;
```

Comment:
I'n mot sure at all about this.

Reply:
Qu'est ce qu'on est censé recevoir ? Un boolean, une string ? La liste des features flags de TcParams ? Si c'est le dernier cas, on sera sur un Record<string, boolean> qui correspond à un object avec une clé en string et une valeur en boolean!

Reply:
Yup, c'est bien le TCParams en entier qui est fourni donc `featureFlags?: Record<string, boolean | undefined>`

Reply:
Niquel, je met ca en type. Merci :)